15314f1ddb882700dc9fabc5ca961943/update/sys_script_include_b630a09ddb9020d01206fd641d9619df.xml

```diff
@@ -718,20 +718,355 @@ IllumioUtils.prototype = {
             found: false
         };
     },
+	
+	/**
+	 * Persist records of  the provided configuration
+	 * @param {object} GlideRecord object of PCE configuration
+	 */
+	persistLabelDimensions: function(pceConfig) {
+		var labelDimensionGr = new GlideRecord("x_illu2_illumio_illumio_label_dimension");
+		labelDimensionGr.addQuery("illumio_pce_configuration", pceConfig.getUniqueValue());
+		labelDimensionGr.addNotNullQuery("href");
+		labelDimensionGr.query();
+		labelDimensionGr.setValue("isdeleted", false);
+		labelDimensionGr.setWorkflow(false);
+		labelDimensionGr.updateMultiple();
+	},
+	
+	/**
+	 * Make REST API call to Illumio PCE
+	 * @params {String} the name of the RestMessage to be used
+	 * @params {String} the name of the HTTP Method to be used
+	 * @parmas {object} string parameters for the RESTMessage
+	 * @params {object} headers to be passed to the RestMessage
+	 * @params {object} body to be passed to the RestMessage
+	 * @params {String} the MID Server to be used
+	 * @returns {object} the response object returned by the API
+	 */
+	makeRestCall: function(restMessage, httpMethod, stringParams, headers, body, midServer) {
+		var i;
+		var apiCall = new sn_ws.RESTMessageV2(restMessage, httpMethod);
+		if (!gs.nil(stringParams)) {
+			Object.keys(stringParams).forEach(function(key) {
+				apiCall.setStringParameter(key, stringParams[key]);
+			});
+		}
+		if (!gs.nil(headers)) {
+			Object.keys(headers).forEach(function(key) {
+				apiCall.setRequestHeader(key, headers[key]);
+			});
+		}
+		if (!gs.nil(body) && Object.keys(body).length > 1) {
+			apiCall.setRequestBody(JSON.stringify(body));
+		}
+		if (!gs.nil(midServer)) {
+			apiCall.setMIDServer(midServer);
+			apiCall.setEccParameter("skip_sensor", "true");
+		}
+		return apiCall.executeAsync();
+	},
+	
+	/**
+	 * Hit Illumio PCE API to get the label dimensions for a provided organisation
+	 * @param {String} pceConfig object of configuration
+	 * @returns {dict} Dict containing the status, response and response status code
+	 */
+	getLabelDimensions: function(pceConfig) {
+		try {
+			var returnDict = {
+				"status": false,
+				"response": "",
+				"statusCode": -1
+			};
+
+			if (!(pceConfig.mid_server.status == MID_SERVER_UP_STATUS && pceConfig.mid_server.validated == 'true')) {
+				gs.error("The configured MID Server with configuration record " + pceConfig.getUniqueValue() + " is down.");
+				returnDict["status"] = false;
```

Comment:
status is already false by default, no need to do this again
can just proceed with returning "returnDict"